src/manifest.json

```diff
@@ -8,14 +8,34 @@
   },
   "children": [
     {
-      "type": "internal-link",
+      "type": "group",
       "id": "redis_use_cases",
-      "label": "Redis Use Cases",
-      "summary": "A collection of several common use cases for Redis.",
+      "label": "Redis basic and RAG use cases",
       "args": {
-        "initialIsOpen": true,
-        "path": "/uc/intro.md"
-      }
+        "initialIsOpen": false
+      },
+      "children": [
+        {
+          "type": "internal-link",
+          "id": "redis_use_cases_basic",
+          "label": "Basic Redis use cases",
+          "summary": "A collection of common use cases for Redis.",
+          "args": {
+            "initialIsOpen": true,
+            "path": "/uc/intro.md"
+          }
+        },
+        {
+          "type": "internal-link",
```

Comment:
I think it makes sense to move this tutorial under the existing "Vector search explained" section and rename it to "Vector search and RAG".

Reply:
Olga wanted it at the top under the general use cases section. So, maybe check with her?

Reply:
@ollopaci , any reason for this?

Reply:
Hi @ViktarStarastsenka and @ollopaci . I don't really have an axe to grind on how the tutorial content is organized, so I'll leave it up to you to decide.